configs/modules.config

```diff
@@ -218,6 +218,7 @@ process {
     withName: 'MINIMAP2_ALIGN_READS' {
         tag        = { meta.assembly.build }
         ext.args   = "-x map-hifi"
+        ext.prefix = { "${meta.id}_" + (reads.getName().endsWith(".gz") ? reads.getBaseName(2) : reads.getBaseName()) } 
```

Comment:
Where did you find this:
```
reads.getBaseName(2)
```

Reply:
I found it here:
https://github.com/nextflow-io/nextflow/blob/daded334b2478596d9821fb9049f91e05fc22699/subprojects/nxf-commons/src/main/nextflow/extension/FilesEx.groovy#L335-L349

Reply:
I learned something new